draft-ietf-quic-multipath.md

```diff
@@ -1382,6 +1382,31 @@ Maximum Path Identifier:
   Receipt of a value that is higher than the local maximum value MUST
   be treated as a connection error of type PROTOCOL_VIOLATION.
 
+
+## PATH_CID_BLOCKED frames {#path-cid-blocked-frame}
+
+A sender SHOULD send a PATH_CID_BLOCKED frame (type=0x15228c0e) when
+it wishes to open a path but is unable to do so because there are no unused connection IDs available
```

Comment:
```suggestion
it wishes to open a path or change the CID on an established path
but is unable to do so because there are no unused connection IDs available
```
I think for CIDs we need to cover this case as well.